app/assets/stylesheets/spotlight/_pages.scss

```diff
@@ -103,9 +103,16 @@
 }
 
 .contact-photo {
-    border: 1px solid #CCC;
-    float: left;
-    margin-right: 15px;
+  border: 1px solid #CCC;
```

Comment:
Color literals like `#CCC` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color `#CCC` should be written as `#ccc`